### PR TITLE
Pal 1063 performance tests fixes

### PR DESCRIPTION
**Pull request by dev930018**
Awaiting merge of PAL-1025

### CODE REVIEW

**Review by developer01189998819991197253 — DISMISSED** [review]

**Comment by ac74475** [comment]
The instructions for an automated deployment of the performance tests do not work as I dont think the service manager deploys the required kafka and redis. Therefore the documentation needs updating to get the user to run the docker compose first to start those services.

**Comment by ac74475** [comment]
Can we also remove all the legacy performance test results from the performance modules readme and make it clear that the sample results are from a single laptop deployment

File: deployment/values.yaml [hunk 1]
```diff
@@ -50,15 +48,16 @@ global:
       value: "eu-west-2"
 
   additionalConfigMaps:
+    application-debug.yaml: |
+      logging.level.uk.gov.gchq.palisade: DEBUG
+    application-nologs.yaml: |
+      logging.level.root: ERROR
```
Comment (developer01189998819991197253) [inline deployment/values.yaml:54]: do we want to leave debug logging in?

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]

File: deployment/values.yaml [hunk 2]
```diff
@@ -50,15 +48,16 @@ global:
...
     application-s3.yaml: |
       resource:
         implementation: s3
       data:
         implementation: s3
 
-  # repository: the name of the docker image repository that is the source of local application images that was populated during the project build process and will now be referenced in the pods
-  repository: "gchq/palisade-"
-
```
Comment (ac74475) [inline deployment/values.yaml:61]: this is required

**Review by developer01189998819991197253 — DISMISSED** [review]

File: deployment/values.yaml (hunk 1 above)
Comment (ac74475) [reply deployment/values.yaml:54]: probably not as default

**Review by ac74475 — CHANGES_REQUESTED** [review]
see comment

**Review by developer01189998819991197253 — APPROVED** [review]

**Review by ac74475 — DISMISSED** [review]

**Pull request merged**